src/resources/packages/classy/types/TicketsApiParams.d.ts

```diff
@@ -0,0 +1,5 @@
+export type TicketsApiParams = {
```

Comment:
Conflate this file and the `TicketsApiResponse.d.ts` file into an `Api.d.ts` one?